tests/resources/db_conf.py

```diff
@@ -123,15 +123,15 @@ def create_conf_db(self):
                 for i in range(1, self.hosts_count + 1):
                     name = f"checkh{i}"
                     cursor.execute(
-                        f"INSERT INTO command (command_name,command_line) VALUES (\"{name}\",\"{ENGINE_HOME}/check.pl --id 0 --state {i}\")")
+                        "INSERT INTO command (command_name,command_line) VALUES (\"{2}\",\"{0}/check.pl --id 0 --state {1}\")".format(ENGINE_HOME, i, name))
```

Comment:
Here again, you remove a good code to replace it by a bad one.

Reply:
done